docusaurus.config.js

```diff
@@ -13,14 +13,17 @@ module.exports = {
   // do not delete the following 'noIndex' line as it is modified for staging
   noIndex: false,
   plugins: [
-    //    ["@edno/docusaurus2-graphql-doc-generator",
-    //      {
-    //        schema: "http://localhost:8080/tasklist/graphql",
-    //        rootPath: "./docs/", // docs will be generated under (rootPath/baseURL)
-    //        baseURL: "apis-clients/tasklist-api",
-    //        linkRoot: "/docs/"
-    //      },
-    //    ],
+//        ["@edno/docusaurus2-graphql-doc-generator",
```

Comment:
should we remove this commented part?

Reply:
No, we use this configuration when generating the docs locally. The instruction is here: https://github.com/camunda-cloud/tasklist/wiki/Generate-GraphQL-API-docs